src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/entity/CertificateTypeDB.kt

```diff
@@ -27,7 +27,7 @@ import jakarta.persistence.*
     uniqueConstraints = [UniqueConstraint(columnNames = ["certificate_type", "certificate_version"])]
 )
 class CertificateTypeDB(
-    @Column(name = "certificate_type", nullable = false)
+    @Column(name = "certificate_type", unique = true, nullable = false)
```

Comment:
Is this unique intended to be herE? There is already a unique constraint for both columns defined

Reply:
Not need i think by mistake it has been putted.
Removed.